site/src/server.rs

```diff
@@ -581,27 +581,29 @@ pub async fn handle_graph(
         })
     }
 
-    let mut by_krate = HashMap::new();
-    let mut by_krate_max = HashMap::new();
+    let mut by_test_case = HashMap::new();
```

Comment:
Should this be `by_benchmark`?

Reply:
Oh, I see. This is a `HashMap<Benchmark, HashMap<Profile, Vec<Scenario>>>`. Nevermind.